hbw/app/javascript/packs/hbw/components/form.js.jsx

```diff
@@ -157,6 +161,10 @@ modulejs.define('HBWForm', ['React', 'jQuery', 'HBWError', 'HBWFormDatetime',
       return false;
     };
 
+    getUpdatedFormValues = ({ name, value }) => {
```

Comment:
You call it `get...` but the only thing it does is `setState` that returns nothing.